### PR TITLE
AC-656 : Rectify the Password seeing option

### PR DESCRIPTION
**Pull request by VibhorChinda**
## Description of what I changed

I have rectified the password seeing option in the password edit text field

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-656

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by VibhorChinda** [comment]
WhatsApp Image 2020-01-16 at 8 30 13 AM (1)

**Comment by VibhorChinda** [comment]
WhatsApp Image 2020-01-16 at 8 30 13 AM

**Comment by VibhorChinda** [comment]
@f4ww4z sir please have a look at the PR.

**Comment by VibhorChinda** [comment]
@f4ww4z sir please have look.

File: openmrs-client/src/main/res/layout/fragment_login.xml [hunk 1]
```diff
@@ -56,7 +57,7 @@
 				android:layout_height="wrap_content"
 				android:gravity="center"
 				android:orientation="vertical"
-				android:visibility="gone">
+				android:visibility="visible">
```
Comment (VibhorChinda) [inline openmrs-client/src/main/res/layout/fragment_login.xml]: @f4ww4z sir added this just to make the whole login fragment's layout visible.
It was in invisible state.

It wont affect anything and would be convenient.
Hope this change is ok

Comment (rishabh-997) [reply openmrs-client/src/main/res/layout/fragment_login.xml]: Please revert this change as it causes the following display by the time it fetches location
WhatsApp Image 2020-02-11 at 7 22 05 PM

Comment (rishabh-997) [reply openmrs-client/src/main/res/layout/fragment_login.xml]: else, everything LGTM !!!

Comment (VibhorChinda) [reply openmrs-client/src/main/res/layout/fragment_login.xml]: @rishabh-997 i reverted it.
Now can you merge ??

**Pull request merged**